index.html

```diff
@@ -2,32 +2,32 @@
 <html lang="en">
 
 <head>
-	<meta charset="utf-8">
-	<title>Orange Inc.</title>
-	<link rel="stylesheet" href="css/styles.css">
-	<script src="js/script.js"></script>
+  <meta charset="utf-8">
+  <title>Orange Inc.</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="js/script.js"></script>
 </head>
 
 <body>
 
-<div class="header">
-	<h1>-todos-</h1>
-	<input id="textbox" type="text" class="inputtextbox">
-	<input id="" class="submitbutton" type="submit" value="Submit" onclick="addItem()">
-</div>	
+  <div class="header">
+    <h1>-todos-</h1>
+    <input id="textbox" type="text" class="inputtextbox">
```

Comment:
use more descriptive id names. You can use descriptive class names as well. Also use 'kebob-casing' for class names. 

`id` should be something like 'todoInput' 
`class` should be something like `todo-input`